apps/marketplace/src/app/landing/landing.component.scss

```diff
@@ -0,0 +1,32 @@
+:host {
+  display: block;
+  height: 100%;
+	background: linear-gradient(-45deg, #ee7752, #e73c7e, #23a6d5, #23d5ab);
+	background-size: 400% 400%;
+	animation: gradient 15s ease infinite;
+}
+
+@keyframes gradient {
+	0% {
+		background-position: 0% 50%;
+	}
+	50% {
+		background-position: 100% 50%;
+	}
+	100% {
+		background-position: 0% 50%;
+	}
```

Comment:
https://i.pinimg.com/originals/1f/df/64/1fdf64e2ede6f751fd668788a6900b6d.gif

Reply:
It's not that trippy but close :p